src/tagging-compliance/resource_list.py

```diff
@@ -1,32 +1,42 @@
-# Copyright (c) 2023, Xgrid Inc, https://xgrid.co
-
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-
-#        http://www.apache.org/licenses/LICENSE-2.0
-
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
 import json
```

Comment:
add license in this file

Reply:
updated 131ca5a9d982dca11cc2842e2da5fe3dcbf159eb